charts/cbcontainers-agent/cbcontainers-agent-chart/example-values.yaml

```diff
@@ -0,0 +1,44 @@
+# accessToken is the API token used by the agent to communicate with the backend
+accessToken: ""
+# accountId is the ID of the organization account
+accountId: ""
+# version is the version of the agent that will be installed
+version: "main"
```

Comment:
I'd prefer to make this empty and require an input. We don't publicly expose the `main` version

Reply:
I agree about `main`. but I think we should have a default (latest) which we will bump manually with each release and release a new version of the chart (with the new default)

Reply:
I changed that to `2.3.1` (current latest)